pages/cart/cart.css

```diff
@@ -0,0 +1,126 @@
+@import url(../../style.css);
+@import url(../product/product.css);
+
+.cart-wrapper {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+    height: 77vh;
+    background-color: var(--secondary-grey);
+
+}
+
+.page-header {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    height: 5rem;
+    color: var(--secondary-light);
+    font-weight: 700;
+    font-size: 1.5rem;
+}
+
+.cart-item-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    margin-left: auto;
+    margin-right: 10rem;
+}
+
+/* --------card--------- */
+.horizontal {
+    flex-direction: row;
+    min-width: 450px;
+}
+
+.horizontal .card-header-info {
+    display: none;
+}
+
+.horizontal .card-img {
+    width: 50%;
+}
+
+.horizontal .card-header-img img {
+    border-top-right-radius: unset;
+}
+
+.card-description {
+    width: 50%;
+}
+.product-qty{
+    margin: 0.5rem 0 0.5rem 0;
+    display: flex;
+    align-items: center;
+    gap: 15px;
+}
+.qty-btn{
+    background-color: var(--secondary-grey);
+    border: none;    
+    border-radius: 50%;
+    padding: 0.1rem 0rem 0.1rem 0rem;
```

Comment:
(nit pick)  you can also use  `padding: 0.1rem 0` for  achieving the same results